src/validator.ts

```diff
@@ -551,11 +568,11 @@ export class RegExpValidator {
     // #region Delegate for Options
 
     private get strict() {
-        return Boolean(this._options.strict || this._uFlag)
+        return Boolean(this._options.strict ?? this._uFlag)
```

Comment:
This change should be revert. I think it's probably a typescript-eslint false positive.

Reply:
I don't think it is 🤔

If `strict` is set to `false`, I don't think we should look into `_uFlag`?

Reply:
Then it should be `this._uFlag ?? this._options.strict` instead?

Reply:
No. We should use the `||` operator instead of the `??` operator.

```suggestion
        return Boolean(this._options.strict) || this._uFlag
```

If `Boolean(this._options.strict)` is true, parse without using Annex B.
That is, parse strictly. For example `/{/` is a syntax error. (`/{/` is not a syntax error in the Annex B specification.)

If `this._uFlag` is true, the unicode flag is used. That is, parse strictly. For example `/{/u` is a syntax error (whether or not you use Annex B).